bentoml/yatai/deployment/aws_utils.py

```diff
@@ -0,0 +1,148 @@
+import logging
```

Comment:
Love all the refactoring and cleanups here in this PR, great job! @mayurnewase

Reply:
Thanks,but that made pr unncessarily large.Wanted to refactor in subsequent pr but there was already lot of code which I could reuse from other deployment ops so did it.